LICENSE.txt

```diff
@@ -187,7 +187,19 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright 2023
+   Author Names:
```

Comment:
Chat GPT suggests it's ok to list many authors in this fashion

Reply:
Alright